modules/apps/style-book/style-book-web/src/main/resources/META-INF/resources/view.jsp

```diff
@@ -9,10 +11,13 @@
 
 <%
 StyleBookDisplayContext styleBookDisplayContext = new StyleBookDisplayContext(request, liferayPortletRequest, liferayPortletResponse);
+
+CETManager cetManager = (CETManager)request.getAttribute(CETManager.class.getName());
+FrontendTokenDefinitionRegistry frontendTokenDefinitionRegistry = (FrontendTokenDefinitionRegistry)request.getAttribute(FrontendTokenDefinitionRegistry.class.getName());
 %>
 
 <clay:management-toolbar
-	managementToolbarDisplayContext="<%= new StyleBookManagementToolbarDisplayContext(request, liferayPortletRequest, liferayPortletResponse, styleBookDisplayContext.getStyleBookEntriesSearchContainer()) %>"
+	managementToolbarDisplayContext="<%= new StyleBookManagementToolbarDisplayContext(request, liferayPortletRequest, liferayPortletResponse, cetManager, frontendTokenDefinitionRegistry, styleBookDisplayContext.getStyleBookEntriesSearchContainer()) %>"
```

Comment:
What do you about move this logic to the `ViewMVCRenderCommand`? It would avoid leak implementation details to the JSP, so the jsp could focus on render logic only. 

In the JSP, We could 

> StyleBookManagementToolbarDisplayContext styleBookManagementToolbarDisplayContext = (StyleBookManagementToolbarDisplayContext) request.getAttribute(StyleBookManagementToolbarDisplayContext.class.getName())

 and then 

> <clay:management-toolbar managementToolbarDisplayContext="<%= styleBookManagementToolbarDisplayContext %>"

Does it make sense?